### DISCUSSION TITLE
Plotting gets stuck halfway through Phase 4 sorting buckets

### DISCUSSION THREAD

**Discussion by drowningblue** [Chia-Network/chia-blockchain#6136] [Support]
I am ripping my hair out on this issue. 
I have not created a successful plot. Plotting runs fine until at phase 4 when it sorts a couple of buckets then stops. The process is still running but doesn't do anything. No errors, nothing. It will run forever if I don't reboot my server. 
 
It's running on Ubuntu Server Headless. I checked permissions, they are fine on all the external drives I have tried. (no WD drives)
It was a clean install, but I have reinstalled both Ubuntu and Chia. The only other thing I have running on the server is Plex. 
i7 8th gen NUC
8 GB RAM
Plotting to 512 Samsung NVME

Anything else I can provide please let me know.

**Comment by tiny0little** [comment]
most likely hardware not suitable for plotting. too little RAM, laptop CPU will die from overheating

**Reply by drowningblue** [reply]
The CPU is an i7 8559U. 8000 PassMark score, 8 threads. Watching the temps it says within normal range. Looking at the min specs for chia the I should be able to do at least one plot. I have another lower-powered Windows NUC with a Pentium and it finishes plots fine. (abet really slow). The i7 gets through phases 1-3 within 8 hours. Phase 4 which if I'm reading properly isn't CPU bound, is the part where the files get compressed and moved to final storage and should be the shortest phase. It shouldn't be slow as it is but I don't think its a hardware issue.

**Reply by tiny0little** [reply]
you definitely can try with it, if there's no other options. I was playing with laptop i7 10th gen to plot, it was over-heating severely and constantly throttling. I guess NUC has a bit better cooling...

but you still need 4GB RAM to plot single k32...

**Comment by BrandtH22** [comment]
@drowningblue , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
